docs/components/concepts/clusters.md

```diff
@@ -6,72 +6,88 @@ description: "Learn more about the clusters available in your Camunda 8 plan."
+The cluster type defines the level of availability and uptime for the cluster.
 
-Once you sign up for a Free Trial, you are able to create one production cluster for the duration of your trial.
+For example:
 
-When your Free Trial plan expires, you are automatically transferred to the Free Plan. This plan allows you to model BPMN and DMN collaboratively, but does not support execution of your models. Any cluster created during your trial is deleted, and you cannot create new clusters.
+- Use a **Basic** cluster for experimentation, early development, and basic use cases not requiring a guaranteed high uptime.
+- Use an **Advanced** cluster for production, with minimal disruption and high uptime.
 
-### Auto-pause
+You can choose from three different cluster types:
 
-Free Trial `dev` (or untagged) clusters are automatically paused eight hours after a cluster is created or resumed from a paused state. Auto-pause occurs regardless of cluster usage.
+| Type                                                                                                                                                                                                           | Basic                                                | Standard                      | Advanced                                            |
+| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :--------------------------------------------------- | :---------------------------- | :-------------------------------------------------- |
+| Usage                                                                                                                                                                                                          | Experimentation, early development, basic use cases. | Production-ready use cases.   | Production with minimal disruption and high uptime. |
+| Uptime Percentage<br/> (Core Automation Cluster<strong>\*</strong>)                                                                                                                                            | 99%                                                  | 99.5%                         | 99.9                                                |
+| Uptime Percentage<br/>(Management Components<strong>\*\*</strong>) &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;                    | 99%                                                  | 99%                           | 99%                                                 |
```

Comment:
Management is something separate from the cluster, so we should not mention it here.

Reply:
Removed row and term definition 👍